examples/model_compress/pruning/mnist_torch.py

```diff
@@ -0,0 +1,145 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+'''
+Examples for level pruner on mnist
+'''
```

Comment:
this is a very simple example right? suggest to rename it to "naive_example_torch.py"

Reply:
do you think "naive_prune_torch" is better?

Reply:
agree